docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

```diff
@@ -262,24 +262,50 @@ that it does not block pop-up windows or create an exception for your {kib} URL.
+|===================================================================================================================================================================================================
+| Detector functions                                                                                                                           | Function description                    | Supported
+| count, high_count, low_count, non_zero_count, low_non_zero_count                                                                             | <<ml-count-functions, Count functions>> | yes   
+| count, high_count, low_count, non_zero_count, low_non_zero_count with summary_count_field_name that is not doc_count (model plot not enabled)| <<ml-count-functions, Count functions>> | yes
+| non_zero_count with summary_count_field that is not doc_count using cardinality aggregation in datafeed config (model plot not enabled)      | <<ml-count-functions, Count functions>> | yes
+| distinc_count, high_distinct_count, low_distinct_count                                                                                       | <<ml-count-functions, Count functions>> | yes         
+| mean, high_mean, low_mean                                                                                                                    | <<ml-metric-mean>>                      | yes         
+| min                                                                                                                                          | <<ml-metric-min>>                       | yes         
+| max                                                                                                                                          | <<ml-metric-max>>                       | yes         
+| metric                                                                                                                                       | <<ml-metric-metric>>                    | yes         
+| median, high_median, low_median                                                                                                              | <<ml-metric-median>>                    | yes         
+| sum, high_sum ,low_sum, non_null_sum, high_non_null_sum, low_non_null_sum                                                                    | <<ml-sum-functions, Sum functions>>     | yes         
+| varp, high_varp, low_varp                                                                                                                    | <<ml-metric-varp>>                      | no         
+| lat_long                                                                                                                                     | <<ml-lat-long>>                         | no         
+| info_content, high_info_content, low_info_content                                                                                            | <<ml-info-content>>                     | no         
```

Comment:
We do plot charts (SMV and Anomaly Explorer) for the `info_content` functions if model plot has been enabled.